CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing Guidelines
+
+Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, correction, or additional
+documentation, we greatly value feedback and contributions from our community.
+
+Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
+information to effectively respond to your bug report or contribution.
+
+## Reporting Bugs/Feature Requests
+
+We welcome you to use the GitHub issue tracker to report bugs or suggest features.
+
+When filing an issue, please check [existing open](https://github.com/aws/aws-vsts-tools/issues), or [recently closed](https://github.com/aws/aws-vsts-tools/issues?q=is%3Aissue+is%3Aclosed), issues to make sure somebody else hasn't already
+reported the issue. Please try to include as much information as you can. Details like these are incredibly useful:
+
+-   A reproducible test case or series of steps
+-   Your version of VSTS (if known) and if it is hosted or in house
+-   Anything unusual about your environment
+
+## Building
+
+1. Clone the repo or your fork with `git clone`
+2. Change to the directory you cloned to, and run `npm install`
+3. At this point you can:
+  - Build for testing purposes. Run `npm run fullBuild`
+  - Package for installation into VSTS. Run `npm run fullBuild publisher=<your-publisher-id-here>` VSTS uses this publisher ID to determine if a plugin can be installed, so make sure it matches the one in your VSTS instance.
+
+## Contributing via Pull Requests
+
+Contributions via pull requests are much appreciated. Before sending us a pull request, please ensure that:
+
+1. You are working against the latest source on the master branch.
+2. You check existing open, and recently merged, pull requests to make sure someone else hasn't addressed the problem already.
+3. You open an issue to discuss any significant work - we would hate for your time to be wasted.
+
+To send us a pull request, please:
+
+1. Fork the repository.
+2. Modify the source; please focus on the specific change you are contributing. Please follow the instructions in building to install the hook for prettier to make sure your code is formatted like the repo.
```

Comment:
Consider moving `Please follow the instructions in building to install the hook for prettier to make sure your code is formatted like the repo.` to a sub bullet (2a?) and reword it a little. Perhaps:
`To ensure your code formatting is consistent with the repo, please follow the steps in the **Building** section. This installs a code formatter.`

Reply:
Changed the wording to 
"Please follow the instructions in **Building** to ensure your changes will be formatted properly (`npm install` installs a code formatter)"